src/app/register/supplier/FormComponent.tsx

```diff
@@ -84,13 +110,14 @@ const FormComponent = () => {
     }).catch((error)=> {
         console.log(error)
         //handleToast error in login
-    })
+    });
     setLoading(false);
   };
 
-  const StepOne : any = (props : any) => {
+  const StepOne = (props : any ) => {
```

Comment:
any?
```suggestion
  const StepOne = (props : Partial<IRegisterAccount> ) => {
```

Reply:
they recieve key, next and data.. i wil create they typology